### PR TITLE
add missing TiKV configuration items

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

- Translate from https://github.com/pingcap/docs/pull/7182
- Fix the wrong default value of `whole-key-filtering`
- Fix some format

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7182 https://github.com/pingcap/docs/pull/12665
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -499,6 +554,30 @@ I/O rate limiter 相关的配置项。
...
++ 默认值：`["127.0.0.1:2379"]`
+
+### `retry-interval`
+
++ 初始化 PD 连接时的重试间隔
++ 默认值：`"300ms"`
+
+### `retry-log-every`
+
++ 指定 PD 客户端在观察到错误时跳过报告错误的频率。例如，当配置项值为 `5` 时，PD 观察到错误后，将在每 4 次错误时跳过报告，直到每第 5 次错误时才报告。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:571]:
```suggestion
+ 指定 PD 客户端在观察到错误时跳过报错的频率。例如，当配置项值为 `5` 时，每次 PD 观察错误时，将跳过 4 次报错，直到第 5 次错误时才报告。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -499,6 +554,30 @@ I/O rate limiter 相关的配置项。
...
++ 要禁用此功能，请将值设置为 `1`。
++ 默认值：`10`
+
+### `retry-max-count`
+
++ 初始化 PD 连接的最大重试次数
++ 要禁用重试，将该值设置为 `0`。要解除重试次数的限制，将该值设置为 `-1`。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:578]:
```suggestion
+ 要禁用重试，请将该值设置为 `0`。要解除重试次数的限制，请将该值设置为 `-1`。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -611,12 +698,14 @@ raftstore 相关的配置项。
 
 ### `raft-log-gc-count-limit`
 
-+ 允许残余的 Raft 日志个数，这是一个硬限制。默认值为按照每个日志 1MB 而计算出来的 3/4 region 大小所能容纳的日志个数。
++ 允许残余的 Raft 日志个数，这是一个硬限制。
++ 默认值：按照每个日志 1MB 而计算出来的 3/4 Region 大小所能容纳的日志个数
```
Comment (shichun-0415) [inline tikv-configuration-file.md:702]:
```suggestion
+ 默认值：3/4 Region 大小所能容纳的日志个数，按照每个日志 1 MB 计算
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1063,6 +1177,11 @@ rocksdb 相关的配置项。
 + 最小值：0
 + 单位：B|KB|MB|GB
 
+### `rate-limiter-refill-period`
+
++ 控制 I/O 令牌的刷新频率。较小的值可以减少 I/O 突发，但会增加 CPU 开销。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1182]: `突发`好像不太合适

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1116,6 +1235,11 @@ rocksdb 相关的配置项。
 + 日志存储目录。
 + 默认值：""
 
+### `info-log-level`
+
++ RocksDB 的日志级别。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1240]: 句点的使用可以统一下

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1476,10 +1600,131 @@ raftdb 相关配置项。
...
++ 默认值：`"20MB"`
++ 最小值：`0`
++ 单位：B|KB|MB|GB
+
+### `create-if-missing`
+
++ 如果值为 `true`，当数据库不存在时将自动创建。
++ 默认值：`true`
+
+### `enable-statistics`
```
Comment (tabokie) [inline tikv-configuration-file.md:1621]: FYI This is removed in https://github.com/pingcap/docs/pull/11626

**Review by tabokie — APPROVED** [review]

File: tikv-configuration-file.md (hunk 6 above)
Comment (Oreoxmt) [reply tikv-configuration-file.md:1621]: The configuration item is `raftdb.enable-statistics` and https://github.com/pingcap/docs/pull/11626 removes `rocksdb.enable-statistics`

File: tikv-configuration-file.md (hunk 4 above)
Comment (Oreoxmt) [reply tikv-configuration-file.md:1182]:
```suggestion
+ 控制 I/O 令牌的刷新频率。较小的值可以减少 I/O 尖刺，但会增加 CPU 开销。
```

File: tikv-configuration-file.md (hunk 6 above)
Comment (tabokie) [reply tikv-configuration-file.md:1621]: They should both be removed. Seems I forgot the raftdb one.

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tikv-configuration-file.md (hunk 6 above)
Comment (Oreoxmt) [reply tikv-configuration-file.md:1621]: https://github.com/pingcap/docs/pull/12665 PTAL

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -499,6 +554,30 @@ I/O rate limiter 相关的配置项。
...
+
+### `retry-log-every`
+
++ 指定 PD 客户端在观察到错误时跳过报错的频率。例如，当配置项值为 `5` 时，每次 PD 观察错误时，将跳过 4 次报错，直到第 5 次错误时才报告。
++ 要禁用此功能，请将值设置为 `1`。
++ 默认值：`10`
+
+### `retry-max-count`
+
++ 初始化 PD 连接的最大重试次数
```
Comment (TomShawn) [inline tikv-configuration-file.md:577]:
```suggestion
+ 初始化 PD 连接的最大重试次数。
```

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -928,6 +1023,20 @@ coprocessor 相关的配置项。
 + 分裂后新 Region 的 key 的个数，此值属于估算值。
 + 默认值：960000
 
+### `consistency-check-method`
+
++ 指定数据一致性检查的方法
```
Comment (TomShawn) [inline tikv-configuration-file.md:1028]:
```suggestion
+ 指定数据一致性检查的方法。
```

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -928,6 +1023,20 @@ coprocessor 相关的配置项。
...
++ 对于 MVCC 数据的一致性检查，设置该值为 `"mvcc"`。对于原始数据的一致性检查，设置该值为 `"raw"`。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1029]:
```suggestion
+ 要对 MVCC 数据进行一致性检查，设置该值为 `"mvcc"`。要对原始数据进行一致性检查，设置该值为 `"raw"`。
```

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -1476,10 +1600,126 @@ raftdb 相关配置项。
...
++ 默认值：`true`
+
+### `allow-concurrent-memtable-write`
+
++ 控制是否开启并发 memtable 写入。
++ 默认值：`true`
+
+### `bytes-per-sync`
+
++ 异步 Sync 限速速率。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1684]: 和英文表述不太一样

File: tikv-configuration-file.md [hunk 11]
```diff
@@ -1476,10 +1600,126 @@ raftdb 相关配置项。
...
++ 默认值：`"1MB"`
++ 最小值：`0`
++ 单位：B|KB|MB|GB
+
+### `wal-bytes-per-sync`
+
++ WAL Sync 限速速率。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1691]: 和英文表述不太一样

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**